launcher/src/components/UI/staking-page/components/list/rows/KeyRow.vue

```diff
@@ -202,6 +213,72 @@ const getKeyState = computed(() => {
   }
 });
 
+const getKeyHandlingTime = computed(() => {
+  const item = props.item.status;
+  switch (item) {
+    case "active_online":
+    case "active":
+    case "active_offline":
+    case "slashed":
+      return props.item.activeSince;
+    case "pending":
+      return props.item.elgibilitySince;
+    case "exited":
+      return props.item.exitSince;
+    case "withdrawal":
+      return props.item.withdrawableSince;
+    case "NA":
+    case "loading":
+    default:
+      return "-";
+  }
+});
+
+const getKeyHanlingFooter = computed(() => {
```

Comment:
This is probably the same typo i already pointed out

```suggestion
const getKeyHandlingFooter = computed(() => {
```